app/cu_editor.py

```diff
@@ -75,8 +74,8 @@ def __init__(self, host, textBuffer):
     self.document = host
     commandSet = initCommandSet(self, textBuffer)
     commandSet.update({
-      ord('n'): self.closeFile,
-      ord('y'): self.writeOrConfirmOverwrite,
+      'n': self.closeFile,
+      'y': self.writeOrConfirmOverwrite,
```

Comment:
I think these two lines can/should stay as ord() since they are looking for the ascii value.